### PR TITLE
Use array to map column to rules

### PR DESCRIPTION
**Pull request by devNoiseConsulting**
- Mapped the column types to validation rules.
- Changed the code to lookup the validation rule instead of the if else code.
- If validation rule is for a string then the max rule is applied as well.

### CODE REVIEW

File: src/Console/ValidateTableCommand.php [hunk 1]
```diff
@@ -21,6 +21,94 @@ class ValidateTableCommand extends Command
      */
     protected $description = 'Generate a validation array for the specified table';
 
+    /**
+     * The laravel validation rule to map column types to.
+     *
+     * @var array
+     */
+    protected $columnTypeRules = [
```
Comment (ahmetsabri) [inline src/Console/ValidateTableCommand.php]: I liked it thank you, but this array seems too long, could you move it to a json file and resolve it by json_decode() method?
then I will merge it.

Comment (devNoiseConsulting) [reply src/Console/ValidateTableCommand.php]: Moved array data to a json file. Can also be published to the app's resources dir for customization.

Comment (ahmetsabri) [reply src/Console/ValidateTableCommand.php]: I was about to merge it but I made some changes to to .gitignore file  that caused conflicts to your PR, can you please resolve it :)

Comment (devNoiseConsulting) [reply src/Console/ValidateTableCommand.php]: I've fixed the merge conflict. Should be good to merge now.

**Pull request merged**